src/GraphqlQueryStore.ts

```diff
@@ -1,70 +1,70 @@
 /* eslint-disable max-classes-per-file */
 /* eslint-disable class-methods-use-this */
-import { validate, parse, specifiedRules } from 'graphql'
+import { createHash } from 'crypto'
+import { parse, specifiedRules, validate } from 'graphql'
 import { compileQuery, isCompiledQuery } from 'graphql-jit'
 import { createComplexityRule, simpleEstimator } from 'graphql-query-complexity'
-import { createHash } from 'crypto'
 import { GraphqlValidationError } from './errors'
 
 export default class GraphqlQueryStore {
-    constructor(schema, {
-        validationRules = [],
-        queryComplexity: {
-            maximumComplexity = 1000,
-            defaultComplexity = 1,
+    schema: any
```

Comment:
Try to avoid any on Library specific code. If some of the test files have any thats ok. But the lib should have everything typed specifically.

Reply:
In this case. If a user passes something else as a Graphql schema to the constructor he will nog get a warning about this.